site/content/docs/5.0/components/toasts.md

```diff
@@ -132,11 +131,24 @@ Building on the above example, you can create different toast color schemes with
 
 ## Placement
 
-Place toasts with custom CSS as you need them. The top right is often used for notifications, as is the top middle. If you're only ever going to show one toast at a time, put the positioning styles right on the `.toast`.
-
-{{< example class="bg-dark" >}}
-<div aria-live="polite" aria-atomic="true" style="position: relative; min-height: 200px;">
-  <div class="toast" style="position: absolute; top: 0; right: 0;">
+Our toast are dynamically placed according to the position you specified and to their container. The top right is often used for notifications, as is the top middle. If you're only ever going to show one toast at a time. If you have several toasts displayed at once, they will stack automatically.
+
+{{< example >}}
+<form>
+  <div class="form-group mb-3">
+    <label for="selectToastPlacement">Toast placement</label>
+    <select class="form-control mt-2" id="selectToastPlacement">
```

Comment:
We need an initial placeholder option value here :) After we add it, feel free to backport it to the visual test too

Reply:
The toast seems to behave a little bugged since we autohide it. Imo, we should disable that and use a default position value like @XhmikosR suggested. 

Also, the class needs some adjustment:
```suggestion
    <select class="form-select mt-2" id="selectToastPlacement">
```